distribution/docs/src/main/jdocs/content/_quickstart/quickstart-installing-contents.adoc

```diff
@@ -0,0 +1,153 @@
+:title: Installing (Quick Start)
+:type: quickStart
+:status: published
+:summary: Installation of an example instance.
+:order: 00
+
+This quick tutorial will enable install, configuring and using a basic instance of ${branding}.
+
+[NOTE]
+====
+This tutorial is intended for setting up a test, demonstration, or trial installation of ${branding}.
+For more complete installation and configuration steps, see <<_installing,Installing>>.
+====
+
+These steps will demonstrate:
+
+- [*] <<_quick_install_prerequisites,Prerequisites>>.
+- [*] <<_quick_install_of_${branding-lowercase},Quick Install of ${branding}>>.
+- [*] <<_ingesting_sample_data,Ingesting Data>>.
+
+=== Quick Install Prerequisites
+
+These are the basic requirements to set up the environment to run a ${branding}.
+
+==== Hardware Requirements (Quick Install)
+
+* At least 4096MB of memory for ${branding}.
+** This amount can be increased to support memory-intensive applications. See <<jvm-memory-configuration, Memory Considerations>>.
+
+==== Java Requirements (Quick Install)
+
+Set up Java to run ${branding}.
+
+* Install/Upgrade to Java 8 http://www.oracle.com/technetwork/java/javase/downloads/index.html[J2SE 8 SDK]
+** The recommended version is http://www.oracle.com/technetwork/java/javase/8u60-relnotes-2620227.html[8u60] or later.
+** Java Version and Build numbers must contain only number values.
+* Supported platforms are *NIX - Unix/Linux/OSX, Solaris, and Windows.
+* http://www.oracle.com/technetwork/java/javase/downloads/index.html[JDK8] must be installed.
+* The `JAVA_HOME` environment variable must be set to the location where the JDK is installed.
+
+.Setting JAVA_HOME variable (Replace `<JAVA_VERSION>` with the version and build number installed.)
+====
+
+. Determine Java Installation Directory (This varies between operating system versions).
+
+.Find Java Path in *NIX
+----
+which java
+----
+
+.Find Java Path in Windows
```

Comment:
I recommend grouping find/update sections by OS. Like this:
- find windows
- update windows
- verify windows
_________________
- find *nix
- update *nix
-verify *nix